serving/installing-logging-metrics-traces.md

```diff
@@ -1,73 +1,65 @@
 # Monitoring, Logging and Tracing Installation
 
-Knative Serving offers two different monitoring setups: 
-One that uses Elasticsearch, Kibana, Prometheus and Grafana and 
-another that uses Stackdriver, Prometheus and Grafana. See below 
-for installation instructions for these two setups. You can install 
-only one of these two setups and side-by-side installation of these two are not supported.
+Knative Serving offers two different monitoring setups:
+[Elasticsearch, Kibana, Prometheus and Grafana](#Elasticsearch,-Kibana,-Prometheus-&-Grafana-Setup) or [Stackdriver, Prometheus and Grafana](#Stackdriver,-Prometheus-&-Grafana-Setup). You can install only one of these two setups and side-by-side installation of these two are not supported.
 
 ## Elasticsearch, Kibana, Prometheus & Grafana Setup
 
-*If you installed Knative Serving using [Easy Install](../install/README.md#Installing-Knative) guide, 
-skip this step and continue to [Create Elasticsearch Indices](#Create-Elasticsearch-Indices)*
+If you installed the [latest Knative Serving components](../install/README.md#Installing-Knative),
+skip this step and continue to [Create Elasticsearch Indices](#Create-Elasticsearch-Indices)
```

Comment:
Create Elasticsearch Indices section was removed from this document - see the comment below.